packages/android_intent/example/lib/main.dart

```diff
@@ -117,6 +117,16 @@ class ExplicitIntentsWidget extends StatelessWidget {
     intent.launch();
   }
 
+  void _openLinkInDefaultBrowserInSeperatedWindow() {
+    final List<String> flags = <String>[];
+    flags.add('new_task');
+    final AndroidIntent intent = AndroidIntent(
+        action: 'action_view',
+        data: Uri.encodeFull('https://flutter.io'),
+        flags: flags);
```

Comment:
- please add a `,` to before the parentheses end
- this now becomes simpler as you can do the bit operation in Dart